prisma/migrations/20231025203103_delete_unused_images_schedule/migration.sql

```diff
@@ -0,0 +1,23 @@
+-- add 'deleteUnusedImages' job to the prisma hack
+-- see migration 20230522153900_schedule_jobs
+CREATE OR REPLACE FUNCTION create_scheduled_jobs()
+RETURNS INTEGER
+LANGUAGE plpgsql
+AS $$
+DECLARE
+BEGIN
+    INSERT INTO pgboss.schedule (name, cron, timezone) VALUES ('trust', '0 2 * * *', 'America/Chicago') ON CONFLICT DO NOTHING;
+    INSERT INTO pgboss.schedule (name, cron, timezone) VALUES ('auction', '* * * * *', 'America/Chicago') ON CONFLICT DO NOTHING;
+    INSERT INTO pgboss.schedule (name, cron, timezone) VALUES ('earn', '0 0 * * *', 'America/Chicago') ON CONFLICT DO NOTHING;
+    INSERT INTO pgboss.schedule (name, cron, timezone) VALUES ('streak', '15 0 * * *','America/Chicago') ON CONFLICT DO NOTHING;
+    INSERT INTO pgboss.schedule (name, cron, timezone) VALUES ('views', '0 0 * * *', 'America/Chicago') ON CONFLICT DO NOTHING;
+    INSERT INTO pgboss.schedule (name, cron, timezone) VALUES ('rankViews', '* * * * *', 'America/Chicago') ON CONFLICT DO NOTHING;
+    INSERT INTO pgboss.schedule (name, cron, timezone) VALUES ('rankViews', '* * * * *', 'America/Chicago') ON CONFLICT DO NOTHING;
+    INSERT INTO pgboss.schedule (name, cron, timezone) VALUES ('deleteUnusedImages', '* * * * *', 'America/Chicago') ON CONFLICT DO NOTHING;
```

Comment:
Do we need to run image deletion every minute?

Reply:
At first this was a copy-paste error but then I realized that if we want to have a consistent 24 hour delete  threshold for images, it makes sense to run this more instead of once per day since else, the delay may become too large.

For example, if someone uploaded something 23h59min ago when the cleanup job starts, it won't get deleted. And then it will stay for another full 24 hours.

Running the cleanup job every minute might be too much but I imagined it doesn't hurt since it runs a query limited by `WHERE paid = 'f' AND created_at < date_trunc('hour', now() - interval '24 hours')` and there is an index on `created_at`. There is no index on `paid` though which may make sense.

So maybe run this every hour or every 3-12 hours?

Reply:
Every hour seems like the right cadence.

You don't need to truncate the date btw. This works too:

`WHERE paid = 'f' AND created_at < now() - interval '24 hours'`

Reply:
> You don't need to truncate the date btw.

Yes but I like clean cutoffs, lol. I don't like to depend on the randomness of the exact moment something was executed even though it's not really important here.

Reply:
Done in 1825cb6 together with the [other comment](https://github.com/stackernews/stacker.news/pull/576#discussion_r1378094163)